### ISSUE TITLE
Missing instruction for upgrade

### ISSUE THREAD

**Issue by bugsysop** [BDX-town/Mangane#136]
Did we have to _uninstall_ current Mangane version before upgrading?

Or can we run directly on top of the old version:

```
/opt/pleroma/bin/pleroma_ctl frontend install mangane --ref dist --build-url https://github.com/BDX-town/Mangane/releases/latest/download/static.zip
```

Sorry, I missed this point when working previously on the README...

**Comment by Cl0v1s** [comment]
No problem I haven't thought about this case myself ! 
You can simply play the install command !

**Comment by bugsysop** [comment]
I'm working on refactoring the README and the doc, I will make a PR later

**Comment by bugsysop** [comment]
Solved by [ea386b6](https://github.com/BDX-town/Mangane/commit/ea386b6f1ed2f9a75a5b6bc97894d3e251523873)
